.circleci/docker/common/install_base.sh

```diff
@@ -126,7 +128,9 @@ install_centos() {
     opencv-devel \
     sudo \
     wget \
-    vim
+    vim \
```

Comment:
Maybe we need [neovim](https://github.com/neovim/neovim/wiki/Installing-Neovim#install-from-download) :P (jk, jk)